### DISCUSSION TITLE
Cloud Infrastructure Setup

### DISCUSSION THREAD

**Discussion by Darjusch** [KitaPlatzZentrale/kpz#115] [Ideas]
Discussion about the migration of the Infrastructure.
Key points are Costs, Time to migrate ( dev, prod, pipelines ), Design of new Infrastructure

For pricing purposes we can use https://calculator.aws/#/.

**Comment by Darjusch** [comment]
### COST + Infrastructure
Here i calculated a basic setup consisting of Route53, S3 Bucket, CloudFront and a small EC2 instance ( there is even smaller like nano ).
https://calculator.aws/#/estimate?refid=b59ef3d1-61fa-4eea-9a0b-96fbd6584e69&id=4f594047c95c4c477e686f985b9eda1cd48d3d60

This would result in costs of around 5.74 USD. 
While I was generous with the calculations of the EC2 instance i.e 8 hours of demand per day.
 

We can think about getting a cheap nano EC2 instance for our dev setup which would cost us maybe 2,40 USD per Month. 
This would make the separation of our setups easier and probably reduce the errors and problems we will run into.

### SETUP

For the setup we will need to create a nginx reverse proxy that forwards the respective requests to the right API.
-> DEV / PROD

We can run the Scraper as a cronJob on the EC2 instance
Pipelines for deploying a production and development version (watchtower ?)

-----------

**Reply by 5h3rr1ll** [reply]
This sounds good, but when this setup is so cheap I don't understand how we reached monthly over 100€ on hanno's account? Only because of the App Runner service?
